### PR TITLE
[KVMService] Initial KVMService file with grpcServer init

### PR DESCRIPTION
**Pull request by seswarrajan**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	pb "github.com/kubearmor/KVMService/protobuf"
+	"google.golang.org/grpc"
+)
+
+type server struct {
+	pb.UnimplementedKVMServer
+}
+
+func (s *server) RegisterAgentIdentity(ctx context.Context, in *pb.AgentIdentity) (*pb.Status, error) {
+	log.Println("Agent Identity is ", in.Identity)
```
Comment (nyrahul) [inline main.go:19]: @nam-jaehyun , is it ok to use `log` for logging?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	os.Setenv("AGENT_IDENTITY", in.Identity)
```
Comment (nyrahul) [inline main.go:20]: We should not be setting the env on KVMService. This needs to be removed.
We will have multiple agents talking to the same KVMService.

File: main.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	return &pb.Status{Status: 200, ErrorMessage: "nil"}, nil
+}
+
+func main() {
+
+	// Get input flag arguments
+	gRPCPtr := flag.String("gRPC", "32767", "gRPC port number")
```
Comment (nyrahul) [inline main.go:27]: For KVMService, we need to use configmap to keep the configuration.

**Review by nyrahul — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (seswarrajan) [reply main.go:19]: This is temporary Rahul. Just for development. WIll change later

File: main.go (hunk 2 above)
Comment (seswarrajan) [reply main.go:20]: Yes Rahul. currently we have not decided how the identity is stored. This value will be later moved to some DB/variable.

File: main.go (hunk 3 above)
Comment (seswarrajan) [reply main.go:27]: Fine Rahul. Will modify the same.

**Pull request closed without merging**